testing/libupnp/APKBUILD

```diff
@@ -1,30 +1,37 @@
 # Contributor: Francesco Colista <fcolista@alpinelinux.org>
 # Maintainer: Francesco Colista <fcolista@alpinelinux.org>
 pkgname=libupnp
-pkgver=1.6.22
+pkgver=1.6.25
 pkgrel=0
 pkgdesc="Portable Open Source UPnP Development Kit"
-url="http://pupnp.sourceforge.net/"
+url="http://pupnp.sourceforge.net"
 arch="all"
-license="BSD"
+license="BSD-3-Clause"
 depends=
 depends_dev=
 makedepends="$depends_dev"
 install=""
-subpackages="$pkgname-dev $pkgname-doc"
+subpackages="$pkgname-dev"
 source="http://downloads.sourceforge.net/sourceforge/pupnp/${pkgname}-${pkgver}.tar.bz2"
-builddir="$srcdir"/$pkgname-$pkgver
+builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
 	cd "$builddir"
-	./configure --prefix=/usr
-	make || return 1
+	./configure \
+		--prefix=/usr \
+		--enable-ipv6 \
+		--disable-static
```

Comment:
Why `--disable-static`?

Reply:
My mistake. Restored.
Is it OK to keep `--enable-ipv6`?

Reply:
Yes, of course, IPv6 should be enabled. ;)